pkg/monitor/build.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+org: lfedge
+image: eve-monitor
+network: yes
+config:
+  pid: host
+  binds:
```

Comment:
same question here, is all these access needed?

Reply:
@shjala probably not.

Reply:
could you go through them, pick the ones you really need and add a comment above each one that is needed stating why it is needed? it makes creating apparmor profile much easier.

same for [this](https://github.com/lf-edge/eve/pull/4410#discussion_r1827533564).